doc/ref/tags/tag_cache.rst

```diff
@@ -42,7 +42,3 @@ Besides the duration and the cache name the ``{% cache %}`` tag also accepts the
 |if_anonymous|Only cache the block when the current visitor is not logged on (i.e. an     |if_anonymous        |
 |            |anonymous visitor)                                                          |                    |
 +------------+----------------------------------------------------------------------------+--------------------+
-|visible_for |Sets the access control user for rendering the block.  With this you can    |visible_for="world" |
-|            |force to only show public items for logged on users.  Valid values are      |                    |
-|            |“user”, 3, “group”, 2, “community”, 1, “world”, “public”, 0                 |                    |
-+------------+----------------------------------------------------------------------------+--------------------+
```

Comment:
@mworrell Do we completely want to remove this argument or should we perhaps keep the distinction between public (not logged in) and user (logged in)?